packages/augur-sdk/src/event-handlers.ts

```diff
@@ -33,7 +33,7 @@ export interface CompleteSetsSold extends FormattedEventLog {
   numCompleteSets: string;
   marketOI: string;
   fees: string;
-  timestamp: number;
+  timestamp: Address;
```

Comment:
I think this be defined as a string instead of an Address. Currently, Address is the same as a string, but my original thinking was that an Address should be formatted like an Ethereum address. (I don't think there's currently a way to do this in TS, but I could be wrong.) I defined the type Address to make the type clearer in the docs.